### ISSUE TITLE
Service in component shared folder - export in index.ts is incorrect

### ISSUE THREAD

**Issue by esinek** [angular/angular-cli#1480]
> Please provide us with the following information:
> ---------------------------------------------------------------

1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
`Windows 7`

2. Versions. Please run `ng --version`. If there's nothing outputted, please run
in a Terminal: `node --version` and paste the result here:
```
angular-cli: 1.0.0-beta.10
node: 6.2.2
os: win32 x64
```

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
```
ng g component myTest
ng g component my-test/mySettings
ng g service my-test/my-settings/shared/mySettings
```

4. The log given by the failure. Normally this include a stack trace and some
   more information.
`no error returned from CLI`

5. Mention any other details that might be useful.
The exports in the index.ts file at the service level (src/app/my-test/my-settings/shared/index.ts) exports the incorrect file. It should be:
`  export * from './my-settings.service'`
but instead is:
`  export * from './mySettings.service'`

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by esinek** [comment]
same issue when service in myTest component's shared folder:

`ng g service my-test/shared/mySettings`

**Comment by filipesilva** [comment]
Definitely a bug, related to https://github.com/angular/angular-cli/issues/1434

**Comment by Gelio** [comment]
This should be closed, as mentioned by @filipesilva in #1434
